data/data/agent/files/usr/local/bin/agent-gather

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+function gather_journal() {
+	( >&2 echo -n "Gathering node journal" )
+	journalctl -o export > ${ARTIFACTS_DIR}/journal.export
+	( >&2 echo " Done")
+}
+
+function gather_agent_data() {
+	( >&2 echo -n "Gathering agent installation data" )
+	mkdir -p ${ARTIFACTS_DIR}/etc
+	cp -a /etc/assisted ${ARTIFACTS_DIR}/etc/assisted
+	( >&2 echo -n ".")
+	mkdir ${ARTIFACTS_DIR}/etc/containers
+	cp -a /etc/containers/registries.conf ${ARTIFACTS_DIR}/etc/containers/
```

Comment:
For disconnected, the output of "ls -l  /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem" to check the file exists would be useful.